### DISCUSSION TITLE
Radio Paradise Meta-Data Bug When Using App

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#1011] [Fixed Issues]
When using the Android app, I find that the RP meta-data and cover art does not always show when selecting web radio after playing a local file.

I can reproduce this when the cover art and data is displayed by going back to the library and selecting the station again with replace+play. The cover art and album data disappears until an event refreshes the screen.

Can anyone else confirm this as a bug? It does not happen when using Firefox on a desktop.

**Comment by Garrido33** [comment]
Same thing here on my official rpi touchscreen and also on firefox with FIP radio and local files. The radio works with the correct coverage, but the title does not refresh and stays on the previous local file. 

If I change the radio station, it is the name of the previous radio which is now displayed.

**Comment by R101G** [comment]
Still seeing this problem. A change of source used to immediately display the default image until new cover art was found. Now the previous cover art is displayed until new art is loaded or not found. Sometimes the new art is displayed momentarily and reverts to the default until an event refreshes the screen. The UI and display is certainly not as slick as it once was.

**Reply by rern** [reply]
Things are somehow faster than before. A little pause is needed to let the current track stop or playlist cleared properly before switching to another track.

@R101G,
Please try the UPDATE branch.

**Comment by R101G** [comment]
Thanks @rern . It looks good now.

**Comment by R101G** [comment]
Sorry @rern , I might have spoken too soon. The problem is back.

Edit: Was the UPDATE fix included in the following updates?

**Comment by R101G** [comment]
@rern 
I still have this problem when using the app. If I select a RP stream after playing a local file or another web stream, the metadata and cover art does not appear until the page is refreshed.

**Reply by rern** [reply]
@R101G,

Please try the UPDATE branch.

**Comment by Garrido33** [comment]
@rern 

Seems to work well for me, after the 20221123 update AND a reboot.

Thanks @rern.

**Comment by R101G** [comment]
@rern 
Thanks, it works for me too now.. I see the RP package is updated too!
